### PR TITLE
Add custom 'slim' template to circleci slack/notify on fail step

### PR DESCRIPTION
**Pull request by 0xdevalias**
This replaces the `basic_fail_1` template ([ref](https://github.com/CircleCI-Public/slack-orb/blob/master/src/message_templates/basic_fail_1.json)) with a custom template that makes the messages slimmer and less spammy.

### CODE REVIEW

**Review by thecadams — APPROVED** [review]
lgtm!

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,8 +3,49 @@ notify-on-fail: &notify-on-fail
     condition: always
     steps:
       - slack/notify:
-          event: fail
-          template: basic_fail_1
+          event: always
```
Comment (0xdevalias) [inline .circleci/config.yml:6]: This was added to test the notification, we need to remove it once we know it works, and before this PR is merged

Comment (0xdevalias) [reply .circleci/config.yml]: restored in https://github.com/sparkletown/sparkle/pull/786/commits/02cd75177a998f4182269cca99a79d7f55ce85fe

**Comment by 0xdevalias** [comment]
The current build failures on this branch are unrelated to changes here.

@thecadams already has a fix for them on https://github.com/sparkletown/sparkle/pull/776

Once that is merged, we can merge into this and then we should be good to go.

**Pull request merged**